tethysext/atcore/templates/atcore/resource_workflows/report_workflow_results_view.html

```diff
@@ -0,0 +1,234 @@
+      }
+      .tab-data {
+        display: block;
+      }
+    </style>
+  {% endif %}
+  <style>
+    #inner-app-content {
+			overflow:auto;
+		}
+		#app-content {
+		  height:100%;
+		}
+    h4 {
+      font-size: 20px;
+    }
+    .map-view-result {
+      width:100%; height:600px;
+    }
+    .legend-list {
+        list-style: none;
+        padding-inline-start: 5px;
+        transition: height 0.2s ease-out;
+        margin-left: 10px;
+        margin-right: auto;
+    }
+
+    .legend-list-item {
+        margin-top: 5px;
+        margin-bottom: 5px;
+    }
+
+    .legend-list-item .flatmark {
+        display: inline-block;
+        max-width: 310px;
+        overflow: hidden;
+    }
+
+    .legend-item {
+        position: relative;
+    }
+
+    .legend-item p {
+        padding-left: 60px;
+        padding-top: 2px;
+    }
+
+    .legend-item .color-box {
+        width: 40px;
+        height: 10px;
+        display: inline-block;
+        position: absolute;
+        left: 5px;
+        top: 5px;
+    }
+    .modebar{
+      display: none !important;
+    }
+    [contentEditable=true]:empty:not(:focus)::before{
+        content:attr(data-ph)
+    }
+    .add-border {
+      margin-top: 10px;
+      margin-bottom: 40px;
+      padding: 5px;
+      border-radius: 5px;
+      border: 1px solid gray;
+    }
+
+    @media print {
+      .map-view-result {
+        width:700px; height:600px;
+      }
+      div:empty {
+          display: none;
+      }
+      div {
+        border-style: none !important;
+      }
+      body {
+        color-adjust: exact;
+        -webkit-print-color-adjust: exact;
+        transform: scale(1);
+      }
+      div[id*="resultType-"] {
+        padding-top: 20px;
+        page-break-inside: avoid;
+      }
+      .add-border {
+        margin-top: 0;
+        margin-bottom: 0;
+      }
+      #inner-app-content {
+        overflow: visible;
+      }
+      #app-content {
+        overflow: visible;
+      }
+      #btnPrint, #app-header, #app-actions,  #app-actions > *, .results-tabs {
+        display: none;
+      }
+      #app-content-wrapper.show-nav #app-content {
+        padding-right: 0;
+        transform: translate3d(0,0,0);
+        overflow: true;
+      }
+      #app-navigation {
+        display: none;
+      }
+      #app-content-wrapper #app-content {
+        padding-top: 0;
+        overflow: visible;
+      }
+      #app-content-wrapper {
+        overflow-x: visible;
+      }
+      a[href]:after {
+        content: none !important;
+      }
+    }
+  </style>
+{% endblock %}
+
+{% block import_gizmos %}
+  {{ block.super }}
+  {% import_gizmo_dependency plotly_view %}
+  {% import_gizmo_dependency bokeh_view %}
+  {% import_gizmo_dependency datatable_view %}
+  {% import_gizmo_dependency map_view %}
+{% endblock %}
+
+{% block app_content %}
+  {{ block.super }}
+
+  <form id="report_form">
+  <div id="report_workflow_data">
+    {% if has_tabular_data %}
```

Comment:
Is anything in this if statement used/needed? Seems like this was copied over from the condor job view.

Reply:
this is why we need the tabular data. This adds the title, general description and parameters to the top of the report.

Reply:
Ah! Got you. Ok. That makes sense. Thanks for the clarification.